contrib/babelfishpg_tsql/sql/sys_function_helpers.sql

```diff
@@ -10126,4 +10126,33 @@ AS 'babelfishpg_tsql', 'pltsql_get_last_stmt_handle' LANGUAGE C;
 
 CREATE OR REPLACE FUNCTION sys.get_babel_server_collation_oid() RETURNS OID
 LANGUAGE C
-AS 'babelfishpg_tsql', 'get_server_collation_oid';
\ No newline at end of file
+AS 'babelfishpg_tsql', 'get_server_collation_oid';
+
+-- Returns True if an object is dependant upon by other objects. False otherwise. 
+CREATE OR REPLACE FUNCTION sys.has_dependencies(schema_name varchar, object_name varchar) RETURNS bool AS
```

Comment:
This function claims to be generic, but it is not. It will only work for views.

Reply:
As discussed offline, removed these permanent catalog functions in favour of creating a temporary function that serves the same purpose in the upgrade script which will then be dropped at the end. Hopefully the function name & description is more clear in the upgrade script.